### PR TITLE
Issue #956: Add default snippetInterface <async\await> and snippetSyntax for async/await

### PR DESCRIPTION
**Pull request by advance512**
Issue #956

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -2,6 +2,8 @@ Please see [CONTRIBUTING.md](https://github.com/cucumber/cucumber/blob/master/CO
 
 ### [Unreleased](https://github.com/cucumber/cucumber-js/compare/v3.2.1...master) (In Git)
 
+* added "asyncawait" option to the snippetInterface formatting option
```
Comment (charlierudolph) [inline CHANGELOG.md]: How about naming this `async-await`?

Comment (advance512) [reply CHANGELOG.md]: I don't like the idea of putting a "-" character in a CLI option. But, if you prefer to do that - let me know, and I will.

Comment (charlierudolph) [reply CHANGELOG.md]: This isn't a typical cli option: https://github.com/cucumber/cucumber-js/blob/master/docs/cli.md#interface. I'd prefer the dash to make it easier to read.

**Comment by aslakhellesoy** [comment]
Hi @advance512,

Thanks for your making your first contribution to Cucumber, and welcome to the Cucumber committers team! You can now push directly to this repo and all other repos under the cucumber organization! 🍾

In return for this generous offer we hope you will:

* ✅ Continue to use branches and pull requests. When someone on the core team approves a pull request (yours or someone else's), you're welcome to merge it yourself.
* 💚 Commit to setting a good example by following and upholding our [code of conduct](https://cucumber.io/conduct) in your interactions with other collaborators and users.
* 💬 Join the [community Slack channel](http://cucumber.io/support#slack) to meet the rest of the team and make yourself at home.
* ℹ️ Don't feel obliged to help, just do what you can if you have the time and the energy.
* 🙋 Ask if you need anything. We're looking for feedback about how to make the project more welcoming, so please tell us!

On behalf of the Cucumber core team,
Aslak Hellesøy
Creator of Cucumber

**Comment by advance512** [comment]
Thanks, @aslakhellesoy. Will do my best, as I love Cucumber. :)

btw - I cannot see that I was added to the organization.

**Pull request merged**